.github/workflows/ci.yml

```diff
@@ -17,14 +17,14 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Pull Image
-        run: docker pull leocov/arduino-processing:latest
+        run: docker pull ghcr.io/leocov-dev/arduino-processing-image:0.0.6
```

Comment:
i switched where this container is built so that it is easier to maintain. the github repo for this container now publishes directly to github instead of docker hub